modules/account-lib/src/coin/dot/keyPair.ts

```diff
@@ -12,18 +20,42 @@ export class KeyPair extends Ed25519KeyPair {
     super(source);
   }
 
+  /**
+   * Helper function to create the KeyringPair for signing a dot transaction.
+   *
+   * @returns {KeyringPair} dot KeyringPair
+   *
+   * @see https://polkadot.js.org/docs/api/start/keyring
+   */
+  protected createPolkadotPair(): KeyringPair {
+    const secretKey = this.keyPair.prv ? new Uint8Array(Buffer.from(this.keyPair.prv, 'hex')) : undefined;
+    const publicKey = new Uint8Array(Buffer.from(this.keyPair.pub, 'hex'));
+    return createPair({ toSS58: keyring.encodeAddress, type: TYPE }, { secretKey, publicKey });
+  }
+
+  /** @inheritdoc */
   getAddress(format?: AddressFormat): string {
-    throw new Error('Method not implemented.');
+    return this.createPolkadotPair().address;
   }
 
-  getKeys(): DefaultKeys {
-    throw new Error('Method not implemented.');
+  /** @inheritdoc */
+  getKeys(): KeyringPair {
```

Comment:
The object is not the proper return format

Reply:
![Screen Shot 2021-11-17 at 11 06 32 PM](https://user-images.githubusercontent.com/78414250/142257189-d73c601e-7300-4a25-9799-2fff4d8c36b2.png)

Made this change based on this comment, what format should it be as we need the KeyringPair instance as well to sign the transaction